packages/dds/tree/src/feature-libraries/editable-tree/editableTreeContext.ts

```diff
@@ -128,8 +123,11 @@ export interface EditableTreeContext {
 export class ProxyContext implements EditableTreeContext {
     public readonly withCursors: Set<ProxyTarget<Anchor | FieldAnchor>> = new Set();
     public readonly withAnchors: Set<ProxyTarget<Anchor | FieldAnchor>> = new Set();
-    private readonly observer: Dependent;
-    private readonly afterChangeHandlers: Set<(context: EditableTreeContext) => void> = new Set();
+    public get emitter(): IEventEmitter<ForestEvents> {
+        return this.forest.emitter;
+    }
+
+    private readonly eventUnregister: (() => void)[];
```

Comment:
We don't need this to be an array yet, but I presume you are making it an array because that's the general pattern for handling an arbitrary number of subscriptions in a disposable object. I bet we could write a little utility class for this and put it in the events package. You don't need to do that in this PR, I'm just thinking out loud.